README.md

```diff
@@ -143,6 +144,7 @@ lspconfig.tsserver.setup({
         includeInlayParameterNameHintsWhenArgumentMatchesName = false,
         includeInlayFunctionParameterTypeHints = true,
         includeInlayVariableTypeHints = true,
+        includeInlayVariableTypeHintsWhenTypeMatchesName = true,
```

Comment:
I think we should use the default value of `false`.
```suggestion
        includeInlayVariableTypeHintsWhenTypeMatchesName = false,
```

Reply:
🤷